experimental/PlaneCurve/test/runtests.jl

```diff
@@ -196,8 +193,7 @@ end
 end
 
 @testset "ProjPlaneCurve int_multiplicity functions" begin
-    R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"])
-    T, _ = grade(R)
+    T, (x, y, z) = graded_polynomial_ring(QQ, ["x", "y", "z"])
     F = ProjPlaneCurve(T((x^2 + y^2) * (x^2 + y^2 + 2 * y * z)))
     G = ProjPlaneCurve(T((x^2 + y^2) * (y^3 * x^6 - y^6 * x^2 * z)))
```

Comment:
Why are you still explicitly moving the polynomial to T -- it is already in T by definition

Reply:
Simple: I did most of the changes in this file with search & replace and regular expressions. It is much nastier to handle `T(arbitrary_expression)` correctly esp. if there are nested parenthesis. Arguably the code was strange *before* (why did work with the non-graded variables in the first place?) Since @wdecker was never quite happy with it either, I am not sure this code will ever graduate out of the "experimental" directory. So my enthusiasm for putting time into cleaning it up further is limited...

Reply:
ok. Makes sense.